pom.xml

```diff
@@ -49,7 +49,7 @@
     <parent>
         <groupId>org.jahia.modules</groupId>
         <artifactId>jahia-modules</artifactId>
-        <version>8.1.4.0-SNAPSHOT</version>
+        <version>8.1.5.0</version>
```

Comment:
This should be switch to 8.1.4.0 (the relase of the snapshot version)

Reply:
Are you sure? It's for the parent version and there is no 8.1.4.0 version of Jahia